src/main/java/seedu/address/model/person/Gender.java

```diff
@@ -9,7 +9,7 @@
  */
 public class Gender {
 
-    public static final String MESSAGE_CONSTRAINTS = "Gender should be either 'M' or 'F'";
+    public static final String MESSAGE_CONSTRAINTS = "Gender should be either 'M' or 'F' and it should not be blank.";
```

Comment:
We might have to add this in our User Guide parameter details. 
Currently its  `Should only be either M (Male) or F (Female).` Perhaps we could add `Should not be blank` as well just like the other parameters in the parameter details. What do you think? 😄

Reply:
I will add it to the other parameters missing 'should not be blank' in another PR 👍🏻